pydicom/tests/test_filewriter.py

```diff
@@ -1504,7 +1574,7 @@ def test_meta_unchanged(self):
 
         # Conformant
         meta = Dataset()
-        meta.FileMetaInformationGroupLength = 62 # Correct length
+        meta.FileMetaInformationGroupLength = 62  # Correct length
         meta.FileMetaInformationVersion = b'\x00\x01'
         meta.MediaStorageSOPClassUID = '1.1'
         meta.MediaStorageSOPInstanceUID = '1.2'
```

Comment:
As @scaramallion did an extra PR for this one, I'm afraid you will have to revert this one and redo the changes (provided his PR gets merged first, what makes sense). But that would make this PR much cleaner also.